testsupport/tiers/checks.go

```diff
@@ -327,6 +382,20 @@ func rbacEditRoleBinding() namespaceObjectsCheck {
 	}
 }
 
+func userViewRoleBinding() namespaceObjectsCheck {
+	return func(t *testing.T, ns *v1.Namespace, memberAwait *wait.MemberAwaitility, userName string) {
+		rb, err := memberAwait.WaitForRoleBinding(ns, "user-view")
+		require.NoError(t, err)
+		assert.Len(t, rb.Subjects, 1)
+		assert.Equal(t, "User", rb.Subjects[0].Kind)
+		assert.Equal(t, userName, rb.Subjects[0].Name)
+		assert.Equal(t, "edit", rb.RoleRef.Name)
```

Comment:
This is supposed to be `view` and not `edit`, right?
```suggestion
		assert.Equal(t, "view", rb.RoleRef.Name)
```